src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/AuthorizationRepository.kt

```diff
@@ -17,23 +19,22 @@ interface AuthorizationRepository
     fun ensureUserGroupHasPermission(userGroup: String, permission: ReifiedPermission)
     fun ensureUserGroupDoesNotHavePermission(userGroup: String, permission: ReifiedPermission)
     fun getPermissionsForUser(email: String): PermissionSet
-    fun getReportReaders(reportName: String): Map<User, List<UserGroupPermission>>
 }
 
-class OrderlyAuthorizationRepository : AuthorizationRepository
+class OrderlyAuthorizationRepository(private val permissionMapper: PermissionMapper = PermissionMapper()) : AuthorizationRepository
```

Comment:
Moved some methods out of the `AuthRepo, which was getting big, into the `UserRepo`. They need to share some permission mapping logic, hence factoring out mapping into a shared class.